inference-engine/tests/functional/shared_test_classes/src/subgraph/transpose_conv_transpose.cpp

```diff
@@ -0,0 +1,86 @@
+// Copyright (C) 2021 Intel Corporation
```

Comment:
I'd add 'squeeze' into the test name since this operation is sufficient for the problem we are testing here.